test/e2e-setup/bicep/modules/managed-identities.bicep

```diff
@@ -44,9 +75,9 @@ resource keyVault 'Microsoft.KeyVault/vaults@2024-12-01-preview' existing = {
 // C L U S T E R   A P I   A Z U R E   M I
 //
 
-resource clusterApiAzureMi 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-01-31' = {
-  name: '${clusterName}-cp-cluster-api-azure-${randomSuffix}'
```

Comment:
Yeah, if we have just one set per RG and they're not cluster-dependent, let's just make this called `cp-cluster-api-azure`, etc